library/ctr_drbg.c

```diff
@@ -313,8 +312,17 @@ void mbedtls_ctr_drbg_update( mbedtls_ctr_drbg_context *ctx,
         if( add_len > MBEDTLS_CTR_DRBG_MAX_SEED_INPUT )
             add_len = MBEDTLS_CTR_DRBG_MAX_SEED_INPUT;
 
-        block_cipher_df( add_input, additional, add_len );
-        ctr_drbg_update_internal( ctx, add_input );
+        /*
+         * block_cipher_df() and ctr_drbg_update_internal() may return error codes.
+         * If they return error, abort the operation, since this public API
```

Comment:
Grammar - "If they return *an* error, abort the operation, since this *function* does not have a return code, we can't return *an* error code to the caller."